src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/components/Deposit.tsx

```diff
@@ -357,7 +496,12 @@ export const Deposit: React.FC<DepositProps> = ({
       contextDispatch({ type: ThorchainSaversDepositActionType.SET_DEPOSIT, payload: formValues })
       contextDispatch({ type: ThorchainSaversDepositActionType.SET_LOADING, payload: true })
       try {
-        const estimatedGasCryptoPrecision = await getDepositGasEstimateCryptoPrecision(formValues)
+        const estimatedGasCryptoPrecision = isUtxoChainId(chainId)
+          ? bnOrZero(estimatedFeesData?.txFeeCryptoBaseUnit)
+              .div(bn(10).pow(feeAsset.precision))
+              .toFixed()
+          : await getDepositGasEstimateCryptoPrecision(formValues)
```

Comment:
The guts of this not breaking input step either when manually inputing, or resulting in wrong percentages, for assets other than UTXOs